src/screens/Home/Home.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import { View, Text, ActivityIndicator, Button } from "react-native";
+import AsyncStorageManager from "@storage/AsyncStorageManager";
+import styles from "./styles";
+
+const Home = ({ navigation: { replace } }: HomeScreenParamList) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState(true);
+  const storageManager = AsyncStorageManager.getInstance();
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      const userId = await storageManager.getLoginUserId();
+
+      if (userId) {
+        const allUsers = storageManager.getUsers();
+        const loggedInUser = allUsers.find(user => user.id === userId);
+        setUser(loggedInUser || null);
+      }
+
+      setLoading(false);
+    };
+
+    fetchUserData();
+  }, []);
+
+  const handleLogout = async () => {
+    await storageManager.saveLoginUserId("");
+    replace("Login");
+  };
+
+  if (loading) {
+    return (
+      <View style={styles.loadingContainer}>
+        <ActivityIndicator
+          size="large"
```

Comment:
why are you not using loader comp when you have already created that

Reply:
This is just a dummy screen to check redirect of authentication it would be removed later